shapes.go

```diff
@@ -187,6 +187,39 @@ func (p *Proc) Triangle(x1, y1, x2, y2, x3, y3 float64) {
 	)
 }
 
+func (p *Proc) Bezier(x1, y1, x2, y2, x3, y3, x4, y4 float64) {
+	if !p.doStroke() {
+		return
+	}
+
+	var (
+		sp  = p.pt(x1, y1)
+		cp0 = p.pt(x2, y2).Sub(sp)
+		cp1 = p.pt(x3, y3).Sub(sp)
+		ep  = p.pt(x4, y4).Sub(sp)
+	)
+
+	path := func(o *op.Ops) clip.PathSpec {
+		var path clip.Path
+		path.Begin(p.ctx.Ops)
+		path.Move(sp)
+		path.Cube(cp0, cp1, ep)
+		return path.End()
+	}
+
+	stk := op.Push(p.ctx.Ops)
```

Comment:
```suggestion
	defer op.Push(p.ctx.Ops).Pop()
```
and remove the `stk.Pop()` line below.

Reply:
fixed